app/src/main/java/com/wizeline/academy/animations/core/data/repository/HomeRepository.kt

```diff
@@ -32,13 +34,18 @@ class HomeRepository @Inject constructor() {
         1025
     )
 
-    fun fetchMockedImages(): List<Int> {
-        val images = mutableListOf<Int>()
+    fun fetchMockedImages(): List<HomeItem> {
+        val items = mutableListOf<HomeItem>()
         for (i in 1..10) {
-            val randomId = getRandomId()
-            images += randomId
+            val item = createNewHomeItem()
+            items += item
         }
-        return images
+        return items
+    }
+
+    private fun createNewHomeItem(): HomeItem {
+        val randomId = getRandomId()
```

Comment:
is it necessary to store the value in a variable? perhaps you could try return homeItem { imageId = getRandomID()}

Reply:
missed that, thanks!